src/OpenTelemetry.AutoInstrumentation.Native/continuous_profiler.cpp

```diff
@@ -336,7 +336,14 @@ void ThreadSamplesBuffer::StartSample(ThreadID                   id,
 {
     CHECK_SAMPLES_BUFFER_LENGTH()
     WriteByte(kThreadSamplesStartSample);
-    WriteString(state->thread_name_);
+    if (state->thread_name_.empty())
+    {
+        WriteString(trace::ToWSTRING(std::to_string(id)));
+    }
+    else
+    {
+        WriteString(state->thread_name_);
+    }
```

Comment:
Why this change? ThreadId != ThreadName. If it is not available, it should be just empty.
Was it for debugging purpises?

Reply:
I have removed this change in scope of 62c8e065f38bde1c463bd92066395e4a97b57d51
I think that is was important for one of the tests. Backends should handle correctly cases when the thread name is not available. Lack of data is better than putting there obviously wrong data.